less_16.js

```diff
@@ -0,0 +1,95 @@
+// # 1 Reverse str
+// Create a function that reverses str
+
+const reverse = (str) => {
+  
+    const reversestr = str.split('').reverse().join('');
+    console.log(reversestr);
+    return reversestr;
+};
+
+reverse("hello");
+reverse("Oleg");
+reverse("JavaScript");
+
+// ===============================================
+
+// # 2 Count words
+// Create a function that counts words in sentence
+
+const countWords = (str) => {
+ 
+    let counter = str.split(' ').length ;
+    console.log(counter);
+    return counter;
+};
+
+countWords("Hi my name is Oleg");
+countWords("Lorem ipsum dolor sit amet consectetur adipisicing elit.");
+
+// ==============================================
+
+// # 3 Clear whitespace
+// Create a function that clears spaces
+
+const clearWhiteSpace = (str) => {
+
+    let clear = str.split(' ').join('');
+    console.log(clear);
+    return clear;
+
+};
+
+clearWhiteSpace("  1   2  5ty  sgsg ");
+clearWhiteSpace("  dd 2 ty aa sgsg ");
+
+// ==============================================
+
+// # 4  Max length
+// Create a function that cuts str if it's length is bigger then max length
+// Add ... at the end if str is bigger
+
+const cutStr = (str, maxLength) => {
+  
+    if ( maxLength === str.length || str.length < maxLength ) {
+
+        console.log(str);
```

Comment:
use logs to show value that returned from function rather then result in function;